src/main/java/frc/robot/subsystems/gyro/Gyro.java

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+package frc.robot.subsystems.gyro;
+
+import org.littletonrobotics.junction.Logger;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Gyro extends SubsystemBase
+{
+    private final GyroIO                _io;
+    public final GyroIOInputsAutoLogged _inputs = new GyroIOInputsAutoLogged();
+
+    public Gyro(GyroIO io)
+    {
+        _io = io;
+    }
+
+    public void periodic()
+    {
+        _io.updateInputs(_inputs);
+        Logger.processInputs("Gyro", _inputs);
+    }
+}
```

Comment:
The method you previously had for getting the tilt should be added in here. We'll also want methods to get the yaw position and yaw rate:

```suggestion
    }

    public Rotation2d getYawPosition()
    {
        return _inputs.yawPosition;
    }
    
    public double getYawVelocityRadPerSec()
    {
        return _inputs.yawVelocityRadPerSec;
    }
    
    public Rotation2d getRollPosition()
    {
        return _inputs.rollPosition;
    }
}
```